lib/model.js

```diff
@@ -376,7 +376,7 @@ var SSL_VALUES = [
 /**
  * @constant {String} - The default value for `ssl`.
  */
-var SSL_DEFAULT = 'IFAVAILABLE';
+var SSL_DEFAULT = 'NONE';
```

Comment:
@imlucas did we decide to not go the "If Available" route? I wasn't involved in that conversation, but I thought that's what was decided. Can you weigh in?